src/main/controlMod.F90

```diff
@@ -689,7 +698,8 @@ subroutine control_spmd()
 
     ! BGC
     call mpi_bcast (co2_type, len(co2_type), MPI_CHARACTER, 0, mpicom, ier)
-    if (use_cn) then
+    
+    if (use_cn .or. use_fates) then
        call mpi_bcast (suplnitro, len(suplnitro), MPI_CHARACTER, 0, mpicom, ier)
        call mpi_bcast (nfix_timeconst, 1, MPI_REAL8, 0, mpicom, ier)
        call mpi_bcast (spinup_state, 1, MPI_INTEGER, 0, mpicom, ier)
```

Comment:
this is a bug that @olyson identified in another thread, spinup_state should never have had its broadcast blocked when use_fates is true

Reply:
Thanks, Ryan.

Reply:
This looks to me like a full fix for #1879, so I'm marking this PR as fixing this.